test/sync/sync_async_interop.dart

```diff
@@ -37,27 +37,27 @@ void runTests(config.createTestDriver createTestDriver) {
 
     test('sync to async driver works', () async {
       final asyncDriver = driver.asyncDriver;
-      expect(asyncDriver, new isInstanceOf<async_core.WebDriver>());
+      expect(asyncDriver, const isInstanceOf<async_core.WebDriver>());
```

Comment:
FYI: please run the tests since I've seen issues with 'const isInstanceOf ' not behaving identical to 'new isInstanceOf'